### PR TITLE
fix: focus on composer new appointment input during appointment creation

### PR DESCRIPTION
**Pull request by FrancescoGottardi**

### CODE REVIEW

**Review by dhavaldodiya — APPROVED** [review]

File: src/view/editor/parts/editor-title.tsx [hunk 1]
```diff
@@ -49,12 +49,14 @@ export const EditorTitle = ({ editorId }: { editorId: string }): ReactElement |
 
 	return !isNil(title) ? (
 		<Input
-			label={t('label.event_title', 'Event title')}
+			label={`${t('label.event_title', 'Event title')}*`}
```
Comment (nubsthead) [inline src/view/editor/parts/editor-title.tsx:52]: Why do you add an asterisk inside the title?

**Review by nubsthead — CHANGES_REQUESTED** [review]

**Review by gnekoz — APPROVED** [review]

File: src/view/editor/parts/editor-title.tsx (hunk 1 above)
Comment (FrancescoGottardi) [reply src/view/editor/parts/editor-title.tsx:52]: the field is now required and start empty, i interpolated it insted of putting the asterisk inside the translation.
Should i change the implementation?

Comment (nubsthead) [reply src/view/editor/parts/editor-title.tsx:52]: Oh ok, I didn't notice it was a desiderata of the task

**Review by nubsthead — APPROVED** [review]

**Review by gnekoz — APPROVED** [review]

**Review by nubsthead — CHANGES_REQUESTED** [review]
Implement the mandatory code to set to false the editor auto focus

**Pull request closed without merging**